lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -116,6 +116,7 @@ def _timer_set_interval(self):
 
 class FigureCanvasWebAggCore(backend_agg.FigureCanvasAgg):
     _timer_cls = TimerTornado
+    supports_blit = False
```

Comment:
Do you want to add a comment that this is a workaround, or is this tracked in an issue and planned to be fixed soon after 3.4 release?